readme.md

```diff
@@ -57,26 +57,51 @@ It seems like B21 (and maybe other models?) has two bluetooth adresses. For me,
 
 ## Examples
 
-**B21, USB connection, 30x15 mm (240x120 px) label**
+### B21, USB connection, 30x15 mm (240x120 px) label
 
 ```
 python niimprint -c usb -a /dev/ttyACM0 -r 90 -i examples/B21_30x15mm_240x120px.png
 ```
 
 [![](examples/B21_30x15_result.png)]()
 
-**B21, Bluetooth connection, 80x50 mm (640x384 px) label**
+## B21, Bluetooth connection, 80x50 mm (640x384 px) label
 
 ```
 python niimprint -c bluetooth -a "E2:E1:08:03:09:87" -r 90 -i examples/B21_80x50mm_640x384px.png
 ```
 
 [![](examples/B21_80x50_result.png)]()
 
-**D11**
+### D11
 
 Completly untested yet, however original fork supports it. If you have D11 at hand and willing to test, please open an issue!
 
+### D110
+
+Pair device that starts from `02:13:` (identifies itself as printer).
+
+`python niimprint -m d110 -c bluetooth -a "02:13:FF:FF:FF:FF" -d 3 -r 90 -i examples/D110/96x172.png`
+
+| Tape type   | Printable dimensions (mm) | Pixels | Layout file       | Notes        |
+|-------------|---------------------------|--------|-------------------|--------------|
+| T14*22      | 13.5×21.5                 | 96×172 | `D110/96x172.png` |              |
```

Comment:
Should probably make a table of supported printers also. Table of labels is a good idea, maybe we can extend it with links to official label catalog (if there is a one?) and RFID tag values (some initial info here: https://github.com/kjy00302/niimprint/issues/4)